app/scripts/modules/kubernetes/src/serverGroup/configure/configuration.service.js

```diff
@@ -16,7 +16,8 @@ module.exports = angular.module('spinnaker.serverGroup.configure.kubernetes.conf
     function configureCommand(application, command, query = '') {
 
       // this ensures we get the images we need when cloning or copying a server group template.
-      let queries = command.containers
+      const containers = command.containers.concat(command.initContainers);
```

Comment:
`command.containers.concat(command.initContainers || []);` would make me feel a little better, since I'm not 100% sure that `initContainers` will always be initialized as an empty array e.g., for a clone operation.